packages/component-base/src/tabindex-mixin.d.ts

```diff
@@ -3,27 +3,32 @@
  * Copyright (c) 2021 Vaadin Ltd.
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
-import { DisabledMixin } from './disabled-mixin.js';
+import { Constructor } from '@open-wc/dedupe-mixin';
+import { DisabledMixinClass } from './disabled-mixin.js';
 
 /**
- * A mixin to provide the `tabindex` attribute.
+ * A mixin to toggle the `tabindex` attribute.
  *
- * By default, the attribute is set to 0 that makes the element focusable.
+ * The attribute is set whenever the user activates the element by a pointer
+ * or presses an activation key on the element from the keyboard.
  *
- * The attribute is set to -1 whenever the user disables the element
- * and restored with the last known value once the element is enabled.
+ * The attribute is removed as soon as the element is deactivated
+ * by the pointer or by releasing the activation key.
```

Comment:
Was the description changed intentionally? The new description doesn't fit the purpose of `TabindexMixin`.

Reply:
Alright, it is already fixed. 👍